docs/README.md

```diff
@@ -27,6 +27,11 @@ in some cases:
     $ sudo gem install jekyll
     $ sudo gem install jekyll-redirect-from
 
+On `macOS`, with the default ruby,
+
+    $ sudo gem install jekyll bundler
+	$ sudo gem install jekyll-redirect-from
```

Comment:
1. Sorry about the indentation issue - forgot to `untabify`. 
2. It seems that without using the `bundler` version, `jekyll` fails to find `pygments`.